airbyte-cdk/java/airbyte-cdk/db-destinations/src/main/kotlin/io/airbyte/cdk/integrations/destination/jdbc/JdbcBufferedConsumerFactory.kt

```diff
@@ -65,9 +68,16 @@ object JdbcBufferedConsumerFactory {
         typerDeduper: TyperDeduper,
         dataTransformer: StreamAwareDataTransformer = IdentityDataTransformer(),
         optimalBatchSizeBytes: Long = DEFAULT_OPTIMAL_BATCH_SIZE_FOR_FLUSH,
+        parsedCatalog: ParsedCatalog? = null
```

Comment:
can we just remove the ConfiguredAirbyteCatalog from here? Looks like we're only using it to get the streams, which seem to also be present in the ParsedCatalog. Am I missing somethin?

Reply:
probably yes! there's a lot of work to wipe out all the v1 code. Didn't want to do that as part of this PR though.

(same with `createTableV1`, etc - I think mysql+mssql are literally the last two destinations using v1 stuff, after which we can nuke all of it)